introduction.md

```diff
@@ -4,8 +4,9 @@ is
 my test 
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?